components/discount/registerDiscount.tsx

```diff
@@ -40,6 +40,15 @@ import {
   getDomainPriceAltcoin,
   getTokenQuote,
 } from "../../utils/altcoinService";
+import { getFreeDomain } from "@/utils/campaignService";
+
+export type GetCustomCalls = (
```

Comment:
no need for a type. 

Use the `Call` type and follow what we do in `registrationCalls.ts`